google/cloud/storage/async_client.cc

```diff
@@ -29,6 +31,8 @@ AsyncClient::AsyncClient(Options options) {
   background_ = MakeBackgroundThreadsFactory(options)();
   connection_ = storage_internal::MakeAsyncConnection(background_->cq(),
                                                       std::move(options));
+  connection_ =
+      storage_internal::MakeTracingAsyncConnection(std::move(connection_));
```

Comment:
Nest the calls rather than re-initializing the member variable?

Reply:
Done